cetmix_tower_server/views/cx_tower_file_template_view.xml

```diff
@@ -6,6 +6,21 @@
         <field name="model">cx.tower.file.template</field>
         <field name="arch" type="xml">
             <form>
+                <header>
+                    <button
+                        name="toggle_active"
```

Comment:
No need to add buttons for that. There is a standard option in the "Actions"  menu for that